services/web/server/src/simcore_service_webserver/director_v2.py

```diff
@@ -47,9 +47,16 @@ async def _request_director_v2(
     session = get_client_session(app)
     try:
         async with session.request(method, url, headers=headers, json=data) as resp:
-            payload: Dict = await resp.json()
             if resp.status >= 400:
+                # in some cases the director answers with plain text
+                payload: Union[Dict, str] = (
+                    await resp.json()
+                    if resp.content_type == "application/json"
+                    else await resp.text()
+                )
```

Comment:
This is an edge case where **director-v0** fails 5xx error, the replay from the director was not in json format. Had to do this to avoid further issues and to allow for better error propagation. Otherwise replies would get lost mid way.

Reply:
Also this fix might also be required in other places.

Reply:
hmm no I think the fix if needed must be in the director-v2 it should always return json

Reply:
could you point me on how to reproduce the issue?

Reply:
Just remove the docker registry when starting the integration test suite for this module and you will get the exact same error.